app/controllers/helpers_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class HelpersController < ApplicationController
+  before_action :set_initiative, only: %i[index create destroy]
+
+  def index
+    render json: @initiative.helpers
+  end
+
+  def create
+    @initiative.helpers << helpers_from_body
+    if @initiative.save
+      render json: @initiative.helpers, status: :created, location: @initiative
+    else
+      render json: @initiative.errors, status: :unprocessable_entity
+    end
+  end
+
+  def destroy
+    @initiative.helpers.delete(Wyeworker.find(params[:id]))
+    if @initiative.save
+      render json: @initiative.helpers, status: :no_content, location: @initiative
```

Comment:
Seems weird to me return a no content status code and a non empty response body, this should probably be an :ok response code

Reply:
Addressed!